qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -128,6 +128,7 @@
   "explorerConfig": [
     {
       "tabTitle": "Datasets",
+      "charts": {},
```

Comment:
Why this empty block?

Reply:
if not specified, there is a console.error about GuppyDataExplorer expecting a prop and getting undefined

Reply:
Oops... should fix that. Thanks